bughouse_console/src/secret_database.rs

```diff
@@ -144,6 +187,19 @@ where
         )
         .execute(&self.pool)
         .await?;
+        sqlx::query(
+            format!(
+                "CREATE TABLE IF NOT EXISTS sessions (
+                    session_id TEXT NOT NULL PRIMARY KEY,
+                    user_name TEXT,
```

Comment:
I would rather use user ID in order to accommodate the potential for username change in the future.

Reply:
There's this unfortunate thing:

```
The parent key is the column or set of columns in the parent table that the foreign key constraint refers to. This is normally, but not always, the primary key of the parent table. The parent key must be a named column or columns i`n the parent table, not the [rowid](https://www.sqlite.org/lang_createtable.html#rowid).
```

I think we should forego using the `rowid` mechanism and spell out the autoincrement thing for all sql implementations. That'll mean a bit of a migration.